dev/format_code.sh

```diff
@@ -63,6 +70,57 @@ fi
 
 COUNTER=0
 
+# this function searches for all files that are processable in directory $1 with
+# extensions with mask that follows. It searchs sorted for tracked files first,
+# then following untracked files
+#
+# All found files are put on stdout so it is parsable by foreach loop
+search_files() {
+  DIRECTORY=$1
+  shift
+
+  MASKS=""
+  while (($#)); do
+    MASKS+=" ${DIRECTORY}**$1"
+    shift
+  done
+
+  if [ "$ALL" == "--all" ]; then
+    TRACKED=$(git ls-files -- $MASKS)
+  else
+    # Only files which differ from the master branch
```

Comment:
```suggestion
    # Only files which differ from the base branch
```

(or any other way to make the statement more accurate)